.moban.dt/Description.jj2

```diff
@@ -0,0 +1,16 @@
+Package: coala
+Version: 0.12.0.dev
+Title: R dependencies of Coala package
+Author: The coala developers
+Mainainer: Lasse Schuirmann, Fabian Neuschmidt, Mischa Krxfcger
```

Comment:
`Mainainer` is spelling mistake; and the value needs to come from the variables in `.moban.yaml`